README.md

```diff
@@ -7,7 +7,11 @@ This project contains the source code for the yiiframework.com Website.
 ## INSTALLATION
 
 Before you start, make sure you have installed [composer](https://getcomposer.org/) and [Node.js](http://nodejs.org/).
- 
+
+You also need to install [Ruby](https://www.ruby-lang.org/en/) and [Sass](http://sass-lang.com/) because the current tool-chain uses Gulp-ruby-sass. Both because it is faster than node-sass, but also because node-sass tends to lag behind Ruby sass.
```

Comment:
Feature-wise it lags a bit behind but not so much now :)
I think it is faster to use Sass from Ruby instead of a Sass emulator implemented in Node.js, but I will investigate.